Cross build with Qt6 is broken without this, since host tools are not
found (tries to use binaries from target dir instead)

--- a/CMakeLists.txt	2023-06-01 08:38:35.000000000 -0300
+++ b/CMakeLists.txt	2023-11-02 22:41:50.717285646 -0300
@@ -10,6 +10,8 @@
 
 include(FindPkgConfig)
 
+find_package(Qt6 COMPONENTS Core REQUIRED)
+
 ###
 
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" "${PROJECT_SOURCE_DIR}/cmake/Modules")
